core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/models/BillingFeature.kt

```diff
@@ -1,22 +1,11 @@
 package com.revenuecat.purchases.kmp.models
 
 /**
- * Enum mapping billing feature types
- * Allows for a common interface when calling feature eligibility methods from hybrid SDKs
+ * Enum mapping billing feature types. Currently only used to check for feature availability
+ * in Android's, Play store.
  */
-public expect enum class BillingFeature {
+public enum class BillingFeature {
     SUBSCRIPTIONS,
     SUBSCRIPTIONS_UPDATE,
     PRICE_CHANGE_CONFIRMATION,
 }
-
-/**
- * Name according to Google Play's `@BillingClient.FeatureType`.
- */
-public val BillingFeature.playBillingClientName: String
```

Comment:
Also removed this. This shouldn't be needed by consumers IMO. Also noticed we're missing a bunch of enum values here in `purchases-android`... But we can work on that separately.